Test/Mono.Cecil.Tests/PortablePdbTests.cs

```diff
@@ -368,6 +369,28 @@ public void EmbeddedCompressedPortablePdb ()
 			}, symbolReaderProvider: typeof (EmbeddedPortablePdbReaderProvider), symbolWriterProvider: typeof (EmbeddedPortablePdbWriterProvider));
 		}
 
+		[Test]
+		public void EmbeddedCompressedPortablePdbFromStream ()
+		{
+			TestBinary("EmbeddedCompressedPdbTarget.exe", module => {
```

Comment:
I think we can make this simpler and not create infrastructure for reading modules as in memory. You can undo all the changes in BaseTestFixture, and in this test you can use:

`var bytes = File.ReadAllBytes (GetAssemblyResourcePath ("EmbeddedCompressedPdbTarget.exe"))`

Create the module from that and run the test.

Reply:
Done!